src/vaadin-message-input.js

```diff
@@ -61,6 +61,16 @@ class MessageInputElement extends ElementMixin(ThemableMixin(PolymerElement)) {
           message: 'Message'
         }),
         observer: '__i18nChanged'
+      },
+
+      /**
+       * Set to true to disable this element.
+       * @type {boolean}
+       */
+      disabled: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true
```

Comment:
Tests for these property config options are missing:

1. Test that `disabled` property is `false` by default,
2. Test that setting `disabled` property also sets the attribute.